src/scipp/visualizations/colors.py

```diff
@@ -4,6 +4,17 @@
 
 import numpy as np
 
+STYLE = {
+    'attrs': '#ff5555',
+    'coords': '#c6e590',
+    'data': '#f6d028',
+    'masks': '#c8c8c8',
+    'button': '#bdbdbd49',  # Color for plot control button.
+    'button_selected': '#bdbdbdbb',  # Color for selected plot control button.
+    'header_text': '#111111',  # Color for table header text.
+    'hover': '#d6eaf8',  # Color for hovering on table rows.
+}
```

Comment:
Where is this used? Shouldn't this be part of Plopp?

Reply:
I think this is used in the css style we are injecting in the html repr?

Reply:
Here? https://github.com/scipp/scipp/pull/3254/files#diff-326b1107b97cc4ee3fa7eb1ccadb74e07065d5c0d7e532a0f0c67692602cc3e9R29

Reply:
Can you move those into the CSS. There is no reason to keep them in Python when we can't customise them anyway.

Reply:
Can you point me to where that is? I can't seem to find the right place to put this. Thanks

Reply:
We load the style as a template and format it using this: https://github.com/scipp/scipp/blob/5b12dc349fe8f2086cd1e237f5bb4a04dd2f6a62/src/scipp/html/resources.py#L23
This is no longer needed, I think. So this file https://github.com/scipp/scipp/blob/main/src/scipp/html/templates/style.css.template can be turned into a regular CSS file and all placeholders (e.g. `$data_color`) can bereplaced with the colours you listed here.

Reply:
But then if it is no longer a template, the `data_color` would have to be duplicated in the `colors.py` file (used by e.g. `sc.show`) and in the `style.css` file.

It seems that `button`, `button_selected`, and `hover` are not used in the `style.css.template` and that only `header_text` would be the one that lives only in `style.css`?

Reply:
> It seems that button, button_selected, and hover are not used in the style.css.template and that only header_text would be the one that lives only in style.css?

The buttons are from back when we had custom buttons on plots that looked a little different from the builtin ones. So this can probably be removed.

Do we actually still use colours in CSS? Or can we drop them with the newer simplified table views?

Reply:
I guess the prefix `sc-table` in the css implies it's only used for the html table representation?

Reply:
Grepping for `sc-table` onyl gives me results in css. So the python code never even generates HTML elements with this class. But best double check the implementation before removing anything.